examples/configs/sensors/Mic_Yeti_sens_1.yml

```diff
@@ -0,0 +1,5 @@
+sensor_id: Mic_Yeti_sens_1
+temporal_res: 44100.0
+spatial_res_x: 1024.0
```

Comment:
This is one where I think the "spatial_res" is not intuitive and would prefer a different name. This is actually the chunk size. I.e., The microphone samples at 44.1 kHz but sends us data in 1024 sample chunks for throughout/overhead reasons.

Reply:
Again, this is an automated pull of data from the db. The next PR will handle refactoring the device and sensor class to handle the changes spelled out in metadator's map_database_to_deviceclass() function. IDK if chunk size is part of that renaming, but I will check